packages/terra-dev-site/tests/jest/webpack/loaders/devSiteBootstrap.test.jsx

```diff
@@ -0,0 +1,152 @@
+jest.mock('../../../../src/webpack/loaderUtils/generateNavigationConfig');
```

Comment:
this test file was renamed from devSiteEntry.test.jsx to devSiteBootstrap.test.jsx